maven-artifact/src/main/java/org/apache/maven/artifact/versioning/VersionRange.java

```diff
@@ -202,6 +203,9 @@ private static Restriction parseRestriction(String spec) throws InvalidVersionSp
     }
 
     public static VersionRange createFromVersion(String version) {
+        if (DefaultArtifact.empty(version)) {
```

Comment:
Or should we throw here for null/empty string? If so, which exception?

Reply:
Null as in line 95